phc/util/api_cache.py

```diff
@@ -0,0 +1,69 @@
+import hashlib
+import json
+from pathlib import Path
+from typing import Callable
+
+import pandas as pd
+
+from phc.util.csv_writer import CSVWriter
+
+DIR = "~/_phc/api-cache"
```

Comment:
Why did you choose this dir location?

Reply:
I wanted it to still show up in the notebooks viewer and also in the home directory so it isn't specific on the notebook location. I'm open for other suggestions.

Reply:
Ok. I'm assuming that since this is a cache it is a hidden directory. The convention is to make it a dotfile: `~/.phc/api-cache`.

Reply:
I really still want it to be easily discoverable by the user. What place would you prefer if I don't want to make it a dotfile?